server/game/core/card/propertyMixins/PrintedPower.ts

```diff
@@ -1,8 +1,11 @@
+import { EffectName } from '../../Constants';
+import type { PrintedAttributesOverride } from '../../ongoingEffect/effectImpl/PrintedAttributesOverride';
 import * as Contract from '../../utils/Contract';
 import type { Card, CardConstructor, ICardState } from '../Card';
 
 export interface ICardWithPrintedPowerProperty extends Card {
     readonly printedPower: number;
```

Comment:
I think we should remove this from the interface and make it private in the implementation to avoid confusion. It's not really clear what's the difference.